### PR TITLE
[feat] Make commit message configurable

### LINKED ISSUE #131: Make sanitize commit message configurable
Right now sanitizing to a branch commits with a default message, we want to make this configurable

### PR DESCRIPTION
**Pull request by tohanss**
Fix #131

### CODE REVIEW

**Comment by slarse** [comment]
@tohanss You're failing style checks, and this branch is not up-to-date. Don't forget to sync with upstream before you start working on new stuff.

File: tests/test_ext_commands.py [hunk 1]
```diff
@@ -477,6 +477,22 @@ def test_raises_invalid_repo_if_invalid_repo(
...
+        target_branch = "student-version"
+        commit_message = "This is not the default!"
+
+        # Just split() ruins our commit_message, so we split on
+        # hashtag instead of whitespace
+        run_repobee(
+            f"sanitize#repo#--target-branch#{target_branch}#"
+            f"--repo-root#{fake_repo.path}#"
+            f"--commit-message#{commit_message}".split("#")
+        )
```
Comment (slarse) [inline tests/test_ext_commands.py:490]: Use `shlex.split()` instead

**Comment by slarse** [comment]
Ping @tohanss

**Comment by slarse** [comment]
Ping @tohanss

**Comment by tohanss** [comment]
Locally, master claimed to be up to date but it seemed to have updated anyway. All should be well now!

File: tests/test_ext_commands.py [hunk 2]
```diff
@@ -477,6 +478,24 @@ def test_raises_invalid_repo_if_invalid_repo(
             exc_info.value
         )
 
+    def test_commit_message(self, sanitizer_config, fake_repo):
+        target_branch = "student-version"
+        commit_message = "This is not the default!"
+
+        # Just split() ruins our commit_message, so we split on
+        # hashtag instead of whitespace
```
Comment (slarse) [inline tests/test_ext_commands.py:486]: Outdated comment
```suggestion
```

File: tests/test_ext_commands.py [hunk 3]
```diff
@@ -477,6 +478,24 @@ def test_raises_invalid_repo_if_invalid_repo(
...
+            shlex.split(
+                f"sanitize repo --target-branch {target_branch} "
+                f"--repo-root {fake_repo.path} "
+                f"--commit-message '{commit_message}'"
+            )
+        )
+
+        fake_repo.repo.git.checkout(target_branch)
+        commit = fake_repo.repo.head.commit
+        assert commit.message == (commit_message + "\n")
```
Comment (slarse) [inline tests/test_ext_commands.py:497]: If we don't care about trailing and leading whitespace, I'd do this instead:

```suggestion
        assert commit.message.strip() == commit_message.strip()
```

**Review by slarse — CHANGES_REQUESTED** [review]
Two very minor comments, after that LGTM

**Review by slarse — APPROVED** [review]
LGTM! Nice feature :)

**Pull request merged**